src/core/namespace.c

```diff
@@ -687,7 +709,19 @@ static int mount_private_dev(MountEntry *m) {
         (void) symlink("/run/systemd/journal/dev-log", devlog);
 
         NULSTR_FOREACH(d, devnodes) {
-                r = clone_device_node(d, temporary_mount);
+                if (can_mknod) {
+                        r = clone_device_node(d, temporary_mount, can_mknod);
+                        if (r < 0) {
+                                if (errno != EPERM)
+                                        goto fail;
+
+                                can_mknod = false;
+                        }
+
+                        continue;
```

Comment:
this should be `else continue` rather than just `continue`, no?

Reply:
addressed